tutorials/tts/FastPitch_MixerTTS_Training.ipynb

```diff
@@ -1,609 +1,612 @@
-                "\n",
-                "*NOTE: The sample data is not enough data to properly train a Mixer-TTS. This will not result in a trained Mixer-TTS and is used to just as example.*"
-            ]
-        },
-        {
-            "cell_type": "code",
-            "execution_count": null,
-            "id": "8db3a903",
-            "metadata": {
-                "scrolled": true
-            },
-            "outputs": [],
-            "source": [
-                "!python mixer_tts.py sample_rate=16000 \\\n",
-                "train_dataset=tests/data/asr/an4_train.json \\\n",
-                "validation_datasets=tests/data/asr/an4_val.json \\\n",
-                "sup_data_types=\"['align_prior_matrix', 'pitch']\" \\\n",
-                "sup_data_path={mixer_tts_sup_data_path} \\\n",
-                "phoneme_dict_path=tts_dataset_files/cmudict-0.7b_nv22.07 \\\n",
-                "heteronyms_path=tts_dataset_files/heteronyms-030921 \\\n",
-                "whitelist_path=tts_dataset_files/lj_speech.tsv \\\n",
-                "pitch_mean={pitch_mean} \\\n",
-                "pitch_std={pitch_std} \\\n",
-                "model.train_ds.dataloader_params.batch_size=6 \\\n",
-                "model.train_ds.dataloader_params.num_workers=0 \\\n",
-                "model.validation_ds.dataloader_params.num_workers=0 \\\n",
-                "trainer.max_epochs=3 \\\n",
-                "trainer.strategy=null \\\n",
-                "trainer.check_val_every_n_epoch=1"
-            ]
-        },
-        {
-            "cell_type": "markdown",
-            "id": "a00f8b88",
-            "metadata": {},
-            "source": [
-                "That's it!\n",
-                "\n",
-                "In order to train FastPitch and Mixer-TTS for real purposes, it is highly recommended to obtain high quality speech data with the following properties:\n",
-                "\n",
-                "* Sampling rate of 22050Hz or higher\n",
-                "* Single speaker\n",
-                "* Speech should contain a variety of speech phonemes\n",
-                "* Audio split into segments of 1-10 seconds\n",
-                "* Audio segments should not have silence at the beginning and end\n",
-                "* Audio segments should not contain long silences inside"
-            ]
-        }
-    ],
-    "metadata": {
-        "kernelspec": {
-            "display_name": "Python 3",
-            "language": "python",
-            "name": "python3"
-        },
-        "language_info": {
-            "codemirror_mode": {
-                "name": "ipython",
-                "version": 3
-            },
-            "file_extension": ".py",
-            "mimetype": "text/x-python",
-            "name": "python",
-            "nbconvert_exporter": "python",
-            "pygments_lexer": "ipython3",
-            "version": "3.9.7"
-        }
-    },
-    "nbformat": 4,
-    "nbformat_minor": 5
+ "cells": [
```

Comment:
Would it be possible to fix this file diff? Looks like there is some issue with the indent level, it's hard to tell what's changed here.

Reply:
I just looked into the file. The only diff was renaming path to `g2ps` and `tts_tokenizers` but the whole thing shifted because of how GitHub treats Jupiter diffs.